lib/Log/Any/Adapter/DERIV.pm

```diff
@@ -141,6 +142,21 @@ our %SEVERITY_COLOUR = (
     fatal    => [qw(red bold)],
     critical => [qw(red bold)],
 );
+
+# Define a lookup hash reference for all sensitive data regex patterns to be logged
+our $sensitive_patterns = {
```

Comment:
I don't think we need a global variable here. I suggest to move it into the sub `mask_sensitive` as a local var.

Reply:
ok I will change this